src/components/NavItems/NavItems.tsx

```diff
@@ -21,6 +21,8 @@ export const NavItems: React.FC<NavItemProps> = ({ linkColor, ...props }) => (
       <Anchor
         key={link.id}
         href={link.url}
+        target={link.id !== 2 ? "_blank" : undefined}
```

Comment:
These params can be defined in constant array and this component will be less dependent on the id's in constant array (in case if order in array will be changed). Example [{ id: 1, label: 'Whitepaper', url: '#', target: '_blank',  rel: 'noopener noreferrer' }, { id: 2, label: 'Documentation', url: '#' }, ...]